scripts/list_updatable_packages

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import json
+import os
+from collections import namedtuple
+from pathlib import Path
+from typing import Generator, Iterable
+
+from debian.changelog import Changelog  # type: ignore
+from debian.debian_support import NativeVersion  # type: ignore
+
+import requests
+
+SESSION = requests.Session()
+
+
+Package = namedtuple('Package', ['package_name', 'version', 'gem_name', 'directory'])
+
+
+def find_packages() -> Generator[Package, None, None]:
+    for plugin_rb in Path('plugins/').glob('**/*.rb'):
+        gem_name = plugin_rb.read_text().strip().split()[1].strip("',\"")
+        package_changelog = next(plugin_rb.parent.glob('**/changelog'))
+        with package_changelog.open() as debchangelog:
+            changelog = Changelog(debchangelog)
+            version = NativeVersion(changelog.version).upstream_version
+            yield Package(str(changelog.package), version, gem_name, plugin_rb.parent.as_posix())
+
+
+def latest_version(spec: Package) -> str:
```

Comment:
Is `spec` the right name?

Reply:
Copy paste from your script ;)

Reply:
Well, RPMs have specs. Do Debian packages have specs? ;)

Reply:
No. Honestly, I would have used "package" in both cases. The fact we're pulling data from a spec is IMHO irrelevant ;)